CAAppKitBridge/Source/NSView+CAMethods.m

```diff
@@ -196,6 +198,6 @@ - (BOOL) isOpaque
   return YES;
 }
 
-// TODO:(stjepanbrkicc) Implement custom dealloc
+/* TODO:(stjepanbrkicc) Implement custom dealloc */
```

Comment:
While GNUstep does not have a particular guide to writing TODOs in code, this is the approach in [some other environments](https://google.github.io/styleguide/cppguide.html#TODO_Comments):

    /* TODO(stjepanbrkicc): Implement custom dealloc. */

<sub>Sitenote: Technically, your username on GitHub is something different. But it does not matter, because GNUstep is not supposed to be primarily hosted on GitHub.</sub>